src/WordPress/Blueprints/Runner/Step/UnzipStepRunner.php

```diff
@@ -2,21 +2,32 @@
 
 namespace WordPress\Blueprints\Runner\Step;
 
+use WordPress\Blueprints\BlueprintException;
 use WordPress\Blueprints\Model\DataClass\UnzipStep;
 use WordPress\Blueprints\Progress\Tracker;
 use function WordPress\Zip\zip_extract_to;
 
 class UnzipStepRunner extends BaseStepRunner {
 
+	/**
+	 * Runs the Unzip Step
+	 *
+	 * @param UnzipStep $input Step.
+	 * @param Tracker   $tracker Tracker.
+	 * @return void
+	 * @throws BlueprintException
+	 */
 	public function run(
 		UnzipStep $input,
-		Tracker $progress = null
+		Tracker $tracker
```

Comment:
Let's call it `progress_tracker` to make the purpose apparent. Alternatively, the `Tracker` class could be refactored to `ProgressTracker` – either of these work.

```suggestion
		Tracker $progress_tracker
```